### ISSUE TITLE
Add CONTRIBUTING file

### ISSUE THREAD

**Issue by garrying** [tomeshnet/node-list#16]
Some ideas on what to include:
- Where one can generate their node contribution
- Quick reference of terminology and what is required
- Code of Conduct
- Some kind of privacy disclaimer (Node will be visible on a map on tomesh.net etc.)

**Comment by dcwalk** [comment]
Yes, would be v. important! We also discussed something like a PR template, maybe we could have the prompt about public data etc... embedded in there?

**Comment by darkdrgn2k** [comment]
Agreed, helping a few people with their node placement i found a few deficiencies in the map's generated json.  Perhaps i should revisit it along with instructions on how to add your node.

Also maybe what it means to put your active node there

1) monitoring will take place 
2) crawling will take place to map it
etc
